packages/core/helper-plugin/lib/src/utils/getAPIInnerErrors/index.js

```diff
@@ -0,0 +1,16 @@
+export default function getAPIInnerErrors(error, { getTrad }) {
+  const errorPayload = error.response.data.error.details.errors;
```

Comment:
Will this nested value be always defined?

Reply:
This file actually existed before and was only moved into the helper plugin, so I can reuse it :)